Signal/src/ViewControllers/ConversationView/ConversationViewController.h

```diff
@@ -6,15 +6,20 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NS_ENUM(NSUInteger, ConversationViewAction) {
+    ConversationViewActionNone,
+    ConversationViewActionCompose,
+    ConversationViewActionAudioCall,
+    ConversationViewActionVideoCall,
```

Comment:
I finally got around to cleaning up this "action on open" logic for conversation view.